src/main/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/compute/v2/OpenStackComputePlugin.java

```diff
@@ -50,11 +50,9 @@ public OpenStackComputePlugin(String confFilePath) throws FatalErrorException {
     /**
      * Constructor used for testing only
      */
-    protected OpenStackComputePlugin(Properties properties, LaunchCommandGenerator launchCommandGenerator,
-                                     OpenStackHttpClient client) {
+    protected OpenStackComputePlugin(Properties properties, LaunchCommandGenerator launchCommandGenerator) {
```

Comment:
place the aux methods, like doRequestInstance after the main methods.

Reply:
besides, there are many refactoring the can be done. Like in deleteInstance, to keep our new pattern you should extract the try catch block to a method called doDeleteInstance and call it. The method getRequestBody can also be refactored, the first for loop can be extract to a well named method. updateFlavors() seems to have the same problem.